stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -52,6 +52,9 @@
 public class Coordinator {
 
     public final static File STABILIZER_HOME = getStablizerHome();
+    public final static String WORKING_DIRECTORY = new String(System.getProperty("user.dir"));
+    public final static File UPLOAD_DIRECTORY = new File(WORKING_DIRECTORY, "upload");
+    public final static String UPLOAD_HOME = UPLOAD_DIRECTORY.getAbsolutePath();
```

Comment:
I would not have 2 fields for this; just use UPLOAD_DIRECTORY, and if you need a string version for it, just call 'getAbsolutePath' where needed. And make WORKING_DIRECTORY = file, not a string. So that it is in line with the other directories.